frontend/src/detailedData.jsx

```diff
@@ -90,11 +97,10 @@ function DetailedData({ navigation, route }) {
       })();
     }
   }, [item, value]));
-
   return (
   // Simple example of displaying data based in route
-    <TouchableWithoutFeedback onPress={Keyboard.dismiss}>
```

Comment:
Why did you remove the `TouchableWithoutFeedback`? It is used to close the keyboard on iOS as introduced in #47

Reply:
yeah I talked to Courtney about this, and the keyboard closes without it on iOS and Android for me. I removed it because removing the feedback made the scroll not work

Reply:
Ok yeah that tests correct.